.github/workflows/build_wheels.yml

```diff
@@ -204,12 +204,11 @@ jobs:
         uses: actions/download-artifact@v4
         with:
           path: dist
-      - name: Publish to TestPyPI
+      - name: Publish to PyPI
         uses: pypa/gh-action-pypi-publish@v1.9.0
         with:
           user: __token__
           password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
It looks like the latest version of the docs say not to use this anymore: https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/